### DISCUSSION TITLE
在某些方案下禁用选字框

### DISCUSSION THREAD

**Discussion by ChihSee-Hsie** [rime/home#1032] [Q&A]
Rime 现在支持的语言已经远不止汉语，而涉及各种拼音文字。当某些拼音文字方案（比如用 Rime 打俄文或者希腊文之类，我有时这么用）码表较为简单（用来转写，没有词库）而可能背下来或者多数时候不需要输入框提示时，即使一时出现重码也很快能够确定所需字符，这时选字框似乎就没有那么必要了。

一个例子：俄文字母表对应的拉丁转写，я 是 ya，而 е 是 ye。在敲下第一个字符 y 时输入法也许不能分辨而跳出选字框，但再敲一个字母就能确定。此时选字框似乎就不必出现。另外对于一些熟悉五笔的用户而言他们似乎也有取消输入框的想法，我记得自己看过一个帖子说他将选字框的字体调到很小，从而近似地关闭了选字框。如果有办法能直接关闭选字框的话，那位用户应该就不需要用这样的方法替代了，岂不美哉？

在此预先感谢各位的回复。

**Comment by LEOYoon-Tsaw** [comment]
鼠鬚管可以把顏色調到全透明實現，改代碼加上一個隱藏候選框也不難。樓主有興趣的話試一試吧。

**Reply by ChihSee-Hsie** [reply]
> 

我方才看到了您在鼠须管上做的https://github.com/rime/squirrel/pull/577，是加上了 inline_candidate 选项？佩服佩服。然而一者我并不懂编程（不是计算机专业），二者我用的是小狼毫，悲夫😂️

**Comment by MokOopsing** [comment]
可以做到 我现在就是配置使用首选嵌入到本文框中，再把候选框调没

将margin_x和margin_y调为一个足够小的负数就可以了

**Reply by ChihSee-Hsie** [reply]
试出来了，感谢！而且发现这种 style 可以根据不同的方案单独设置。
